gitbase-web/templates/service.yaml

```diff
@@ -14,6 +14,7 @@ spec:
       targetPort: http
       protocol: TCP
       name: http
+      nodePort: {{ .Values.service.nodePort }}
```

Comment:
shouldn't this have a conditional to only set it if defined?

Reply:
:+1: